### PR TITLE
commit for Forecasting Module

### PR DESCRIPTION
**Pull request by SukanyaJagtapTx**

### CODE REVIEW

File: src/test/java/pageobjects/SalesforceObjects/ForecastingPage.java [hunk 1]
```diff
@@ -65,6 +66,7 @@ public class ForecastingPage {
     public static By doneButton = By.xpath("//*[text()=\"Done\"]");
     public static By forecastsSettingsOption= By.xpath("//li[@aria-level=\"4\"]/child::div[@title=\"Forecasts Settings\"]//a");
     public static By enableForecastsToggleButton= By.xpath("//div[@class='slds-form-element__control']//span[@part='indicator']");
+    public static By ForecastsToggleButton= By.xpath("//div[@class='slds-form-element__control']//label//span[text()='Enable Forecasts']");
```
Comment (snaphray) [inline src/test/java/pageobjects/SalesforceObjects/ForecastingPage.java]: Please follow a pattern and start the variable Name with small letter

**Review by snaphray — CHANGES_REQUESTED** [review]
Please resolve the review comment

File: src/test/java/SalesforceModules/OppurtunitiesUtil.java [hunk 2]
```diff
@@ -415,18 +420,88 @@ public static void clickOnSalesforceCPQ() {
...
+        if (KeywordUtil.isWebElementVisible(OppurtunitiesObject.opportunityStage, "Closed Won Stage Displayed")) {
+            RunCukesTest.logger.log(LogStatus.PASS, HTMLReportUtil.passStringGreenColor("Verify Status: " + stageStatus));
+        } else {
+            RunCukesTest.logger.log(LogStatus.FAIL, "Status is not showing: " + stageStatus);
 
         }
     }
 
 
+    //(OppurtunitiesObject.opportunityStage,"Stage status Closed Won"));
```
Comment (snaphray) [inline src/test/java/SalesforceModules/OppurtunitiesUtil.java]: do we need this line? if not then please remove it

Comment (SukanyaJagtapTx) [reply src/test/java/SalesforceModules/OppurtunitiesUtil.java]: that line is removed.

File: src/test/java/pageobjects/SalesforceObjects/CampaignObject.java [hunk 3]
```diff
@@ -112,6 +149,19 @@ public static By clickShowActionsInContacts(String name){
     public static By DeleteShowActionInContacts(String name){
         return By.xpath("//table[@role=\"grid\"]//tbody//tr//a[@title='"+name+"']//ancestor::th//following-sibling::td[6]//span[text()='Show more actions']");
     }
+
+    public static By clickShowActionsInLeads(String name){
+        return By.xpath("//table[@role=\"grid\"]//tbody//tr//a[@title='"+name+"']//ancestor::th//following-sibling::td[8]//span[text()='Show Actions']");
+    }
+
+    public static By DeleteShowActionInLeads(String name){
```
Comment (snaphray) [inline src/test/java/pageobjects/SalesforceObjects/CampaignObject.java]: Name should start from small letter

File: src/test/java/pageobjects/SalesforceObjects/ForecastingPage.java [hunk 4]
```diff
@@ -65,11 +66,7 @@ public class ForecastingPage {
     public static By doneButton = By.xpath("//*[text()=\"Done\"]");
     public static By forecastsSettingsOption= By.xpath("//li[@aria-level=\"4\"]/child::div[@title=\"Forecasts Settings\"]//a");
     public static By enableForecastsToggleButton= By.xpath("//div[@class='slds-form-element__control']//span[@part='indicator']");
-
-
-    public static By quickFind = By.xpath("//input[@placeholder='Quick Find']");
-
```
Comment (snaphray) [inline src/test/java/pageobjects/SalesforceObjects/ForecastingPage.java:71]: Are these objects not required any more?

File: src/test/resources/features/SalesForceServiceSupport.feature [hunk 5]
```diff
@@ -3,11 +3,11 @@ Feature: salesforce service support demo
...
   Scenario Outline: salesforce service support demo
     Given Read the test data  "<TestData>" from Excel
-    When navigate to Salesforce as "System Admin"
-    And login to Salesforce with "System Admin" credentials
+    When Navigate to Salesforce as "System Admin"
+    And Login to Salesforce with "System Admin" credentials
     Then click on service support button
     Then create account for service support
-    Then create new opportunity and change its status
+#    Then create new opportunity and change its status
```
Comment (snaphray) [inline src/test/resources/features/SalesForceServiceSupport.feature:10]: this line is commented

**Review by snaphray — CHANGES_REQUESTED** [review]
Please resolve the review comments

**Review by snaphray — APPROVED** [review]
Code Looks fine now

**Pull request merged**